### ISSUE TITLE
Prims generated by PointInstancer are invisible when using new SceneIndex pipeline

### ISSUE THREAD

**Issue by robp-sidefx** [PixarAnimationStudios/USD#3011]
It looks like when using `UsdImagingPiPrototypePropagatingSceneIndex`, pointinstancer prototypes now live under the original prototype location (whereas previously they lived under the pointinstancer). I assume as a result, they can now inherit invisibility from a parent scope.

Concretely, when *not* using the Stage Scene Index pipeline, the following USD snippet results in a visible instance of the sphere; when using the new pipeline the instance is invisible.

```
def Scope "prototypes"
{
    token visibility = "invisible"
    def Sphere "sphere"
    {
        double radius = 1
    }
}

def PointInstancer "instancer"
{
    point3f[] positions = [(0, 0, 0)]
    int[] protoIndices = [0]
    rel prototypes = </prototypes/sphere>
}
```
![inst](https://github.com/PixarAnimationStudios/OpenUSD/assets/159376454/a56190cf-107b-44c2-9703-61c8dedcfa5c)

**Comment by jesschimein** [comment]
Filed as internal issue #USD-9468

**Comment by spiffmon** [comment]
Yep, definitely a bug, @robp-sidefx , and sigh, it really is a pain supporting non-encapsulated prototypes ;-P

**Comment by robp-sidefx** [comment]
This still occurs with 24.08 ... is a fix on the radar?

**Comment by pixar-oss** [comment]
Hi Rob -- yes, this is on our radar!  We haven't had a chance to start working on it yet though.  Is this blocking anything for you?

**Comment by robp-sidefx** [comment]
We're renewing our push towards Solaris being a Scene Index Hydra 2 future-embracing platform, and this trips up one of our internal tests. It's not blocking us, but we're just trying to manage our internal expectations as to what dev work we'll need to do, what's still coming in the next few USD releases, and what'll need to be in a "known limitations".

**Comment by robp-sidefx** [comment]
In poking around the source code for the `piPrototypePropagatingSceneIndex`, I got to reading some of the comments at the top of the file and noted that if we change `def Scope "prototypes"` to `over "prototypes"` in the test case, it'll work with Hydra 2. I'm not going so far as to say "all good, close this ticket", but leaving the information here for completeness and future pondering amongst all parties :)

**Comment by tcauchois** [comment]
This is an incredibly easy asset fix, for sure.  The code fix is a nightmare because it means that transform/visibility inherit from prototype root to point instancer, while all other attributes inherit up namespace, precluding most of the drama-free implementations.  I think last time we looked at assets in the wild (both inside/outside Pixar), we decided that *not* fixing this would get us in trouble, so we're continuing to kick at this for 25.05.

**Comment by jesschimein** [comment]
Hi Rob, quick update that this issue is still on our radar but looks like it'll miss 25.05.  Sorry for the delay!  🙇🏻‍♀️

**Comment by robp-sidefx** [comment]
Thanks for the proactive update, it's much appreciated.
